pydata_sphinx_theme/__init__.py

```diff
@@ -250,11 +249,42 @@ def navbar_align_class():
             )
         return align_options[align]
 
+    def generate_google_analytics_script(kind, **kwargs):
```

Comment:
what role does `kind` play here? I don't see it used (if unused, can we just remove and use a single arg for `id`?

Reply:
Thank you for the comment, I was following the pattern when testing things and forgot to remove it. Your comment also made me realise that we don't really need kwargs here since we are passing the id only so thank you for that as well 😄